src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -543,6 +544,9 @@ public void setPostLogoutRedirectUrl(String postLogoutRedirectUrl) {
 
     @DataBoundSetter
     public void setEscapeHatchEnabled(boolean escapeHatchEnabled) {
+        if (FIPS140.useCompliantAlgorithms() && escapeHatchEnabled) {
+            throw new IllegalArgumentException("Escape Hatch cannot be enabled in FIPS environment");
```

Comment:
[`FormException`?](https://javadoc.jenkins.io/hudson/model/Descriptor.FormException.html#%3Cinit%3E(java.lang.Throwable,java.lang.String))

Reply:
I changes to use `FomException` with commit https://github.com/jenkinsci/oic-auth-plugin/pull/418/commits/1e3b125547b3d20c1596b8390fbd0975edf952b1